app/assets/javascripts/think_feel_do_dashboard/participants/_form.js

```diff
@@ -0,0 +1,150 @@
+(function() {
+  'use strict';
+
+  var ParticipantForm = {};
+  var PASSPHRASE_LENGTH = 4;
+
+  function getRandomInt(min, max) {
+    return Math.floor(Math.random() * (max - min)) + min;
+  }
+
+  ParticipantForm.generatePassword = function() {
+    var password = [];
+
+    for (var i = 0; i < PASSPHRASE_LENGTH; i++) {
+      password.push(words()[getRandomInt(0, words().length)]);
+    }
+
+    this.password = password.join(' ');
```

Comment:
In this case, `this` refers to `ParticipantForm`. An object method always binds the object to `this` whether in strict mode or not. So calling `ParticipantForm.generatePassword()` means that the object is `ParticipantForm`.